### DISCUSSION TITLE
%EC%9A%B0%EB%A6%AC%EC%9D%98-%EC%8A%A4%ED%84%B0%EB%94%94-%ED%81%B4%EB%9F%BD-%EC%9E%90%EB%B0%94-%EC%A4%91%EA%B8%891-%EC%B6%9C%EC%82%AC%ED%91%9C/

### DISCUSSION THREAD

**Discussion by giscus** [SungbinYang/SungbinYang.github.io#16] [Comments]
# %EC%9A%B0%EB%A6%AC%EC%9D%98-%EC%8A%A4%ED%84%B0%EB%94%94-%ED%81%B4%EB%9F%BD-%EC%9E%90%EB%B0%94-%EC%A4%91%EA%B8%891-%EC%B6%9C%EC%82%AC%ED%91%9C/

양성빈의 블로그

https://sungbin.kr/%EC%9A%B0%EB%A6%AC%EC%9D%98-%EC%8A%A4%ED%84%B0%EB%94%94-%ED%81%B4%EB%9F%BD-%EC%9E%90%EB%B0%94-%EC%A4%91%EA%B8%891-%EC%B6%9C%EC%82%AC%ED%91%9C/

**Comment by lazyshu** [comment]
덕분에 공부 잘하고 있어요 늘고생해주셔서 감사합니당!!

**Comment by moonnahye** [comment]
매번 더 좋은 방향으로 갈수있게 고민하시는게 느껴집니다. 이번 스터디도 화이팅입니다!

**Comment by Darimi2222** [comment]
'스터디가 숙제처럼 전혀 느껴져서는 안된다.' 이 부분 공감이 가네요. 이번 스터디도 힘내보겠습니다. 😊

**Comment by tdronnie** [comment]
새로운 스케줄이 공부랑 리뷰에 좋은 영향을 주었으면 좋겠네요!
끝까지 집중해서 잘 달려보겠습니다~ 화이팅

**Comment by hmkim199** [comment]
덕분에 항상 더 열심히 하게 되는 것 같습니다!! 파이팅 :)

**Comment by devmeeple** [comment]
비록 참여하고 있지는 않지만 PR이 워낙 많이 올라와서 
잘 지켜보고 있습니다.  (열 발자국 뒤에서)
앞으로 행보도 응원하겠습니다 :)
